app/RouteHandlers/AddDonationHandler.php

```diff
@@ -93,6 +93,15 @@ private function newHttpResponse( AddDonationResponse $responseModel ): Response
 					)
 				);
 				break;
+			case PaymentType::SOFORT:
+				$httpResponse = $this->app->redirect(
+					$this->ffFactory->newSofortUrlGeneratorForDonations()->generateUrl(
```

Comment:
this needs error handling, since sofort API might return an error, which throws a `RuntimeError`